2.hello-kubernetes/README.md

```diff
@@ -67,7 +67,7 @@ export NODE_APP=$(kubectl get svc nodeapp --output 'jsonpath={.status.loadBalanc
 ```
 
 ## Step 4 - Deploy the Python App with the Dapr Sidecar
-Next, let's take a quick look at our python app. Navigate to the python app in the kubernetes sample: `cd samples/2.hello-kubernetes/python` and open `app.py`.
```

Comment:
There is a conflicting change that fixes this comment but does not add the file back.